packages/story-editor/src/components/canvas/frameElement.js

```diff
@@ -236,11 +240,10 @@ function FrameElement({ id }) {
         />
       )}
       <Wrapper
-        ref={elementRef}
+        ref={combinedFocusGroupRef}
         data-element-id={id}
         {...box}
-        // eslint-disable-next-line styled-components-a11y/no-noninteractive-tabindex -- Needed for being able to focus on the selected element on canvas, e.g. for entering edit mode.
-        tabIndex={0}
```

Comment:
Now frame elements are part of a focus group. They no longer need a tab index of 0.